BioSQL/BioSeq.py

```diff
@@ -536,21 +538,21 @@ def __del_seq(self):
 
     seq = property(__get_seq, __set_seq, __del_seq, "Seq object")
 
-    def __get_dbxrefs(self):
+    @property
+    def dbxrefs(self) -> List[str]:
+        """Database cross references."""
         if not hasattr(self, "_dbxrefs"):
             self._dbxrefs = _retrieve_dbxrefs(self._adaptor, self._primary_id)
         return self._dbxrefs
 
-    def __set_dbxrefs(self, dbxrefs):
-        self._dbxrefs = dbxrefs
+    @dbxrefs.setter
+    def dbxrefs(self, value: List[str]) -> None:
+        self._dbxrefs = value
 
-    def __del_dbxrefs(self):
+    @dbxrefs.deleter
+    def dbxrefs(self) -> None:
         del self._dbxrefs
 
-    dbxrefs = property(
-        __get_dbxrefs, __set_dbxrefs, __del_dbxrefs, "Database cross references"
-    )
-
```

Comment:
Looks like a nice bit of property modernisation there 👍

Reply:
Credit goes to @mr-c :)